modules/dnn/src/layers/layers_common_opt_avx.cpp

```diff
@@ -0,0 +1,199 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Please use this name to enable AVX compilation flags: layers_common.avx.cpp

BTW, we have short version of license header.